src/main/java/org/opensearch/knn/index/codec/KNN9120Codec/KNN9120BinaryVectorScorer.java

```diff
@@ -94,14 +100,18 @@ public BinaryRandomVectorScorerSupplier(ByteVectorValues vectorValues) throws IO
         }
 
         @Override
-        public RandomVectorScorer scorer(int ord) throws IOException {
-            byte[] queryVector = vectorValues1.vectorValue(ord);
-            return new BinaryRandomVectorScorer(vectorValues2, queryVector);
+        public void setScoringOrdinal(int node) throws IOException {
+
+        }
+
+        @Override
+        public float score(int node) throws IOException {
+            return 0;
```

Comment:
Why are we setting this to 0 ?

Reply:
Ignore this file, i added to avoid compilation issue from lucene